esp/esp/program/modules/forms/lineitems.py

```diff
@@ -0,0 +1,52 @@
+from django import forms
+
+from esp.accounting.models import LineItemType, LineItemOptions
+from esp.program.models import Program
+
+class LineItemForm(forms.ModelForm):
+    class Meta:
+        model = LineItemType
+        fields = ('text', 'amount_dec', 'required', 'max_quantity')
+        labels = {
+            'text': 'Name',
+            'amount_dec': 'Cost',
+        }
+        help_texts = {
+            'required': 'Should this line item be automatically added for every student?',
+        }
+        widgets = {
+            'text': forms.TextInput(attrs={'placeholder': '(name)'}),
+            'amount_dec': forms.NumberInput(attrs={'placeholder': '(cost)'}),
+        }
+
+OptionFormset = forms.modelformset_factory(
+    LineItemOptions,
+    fields = ('description', 'amount_dec', 'is_custom'),
+    max_num = 1000,
+    extra = 0,
+    labels = {
+        'description': 'Description',
+        'amount_dec': 'Cost',
+        'is_custom': 'Is custom?'
+    },
+    help_texts = {
+        'amount_dec': 'If blank, takes overall cost',
+        'is_custom': 'Should the student be allowed to specify a custom amount for this option?',
+    },
+    widgets = {
+        'description': forms.TextInput(attrs={'placeholder': '(description)'}),
+        'amount_dec': forms.NumberInput(attrs={'placeholder': '(cost)'}),
+    }
+)
+
+class LineItemImportForm(forms.Form):
+    program = forms.ModelChoiceField(queryset=None)
+
+    def __init__(self, *args, **kwargs):
+        cur_prog = kwargs.pop('cur_prog', None)
+        super(LineItemImportForm, self).__init__(*args, **kwargs)
+        progs = LineItemType.objects.exclude(text__in=["Sibling discount", "Program admission", "Financial aid grant", "Student payment"]).values_list('program', flat = True).distinct()
```

Comment:
There's also a "Donation to Learning Unlimited" line item that's autogenerated by the donation module and unfortunately takes tag inputs for the amount options, not line item edits (this should really change) so that line item should also get excluded.

Additionally, there are also "Saturday Lunch" and "Sunday Lunch" line items generated by the SplashInfoCosts module that take tag inputs, so these should also be excluded.